src/PKSim.Presentation/Presenters/Main/BuildingBlockExplorerPresenter.cs

```diff
@@ -181,5 +195,12 @@ private void handleRenamedBuildingBlock(RenamedEvent renamedEvent)
 
          RefreshTreeAfterRename();
       }
+
+      public void Handle(ProjectLoadedEvent eventToHandle)
+      {
+         //This should be here since there are too many ways to open a simulation
+         //and this should be done before any simulation is loaded.
```

Comment:
I would change this comment to read something like:
```
// Ensures that all individuals are loaded before any simulations can be lazy loaded.
```

Reply:
I do not understand this at all.. Why should the presneter be repsonsibiel of loading all individuals. This makes me very scared

Reply:
Maybe in `ProjectMetaDataToProjectMapper`. We know the project metadata version.